### PR TITLE
Article - Maintain Code Quality in Node using ESLint

### PR DESCRIPTION
**Pull request by cyberShaw**
Added an article about linters and how to use ESLint, a popular JavaScript Linter to maintain code quality in projects.

Please review and merge! 😄

### CODE REVIEW

**Comment by hectorkambow** [comment]
Good afternoon @cyberShaw  - just starting a preliminary review on the ESLint article - it's looking great so far - but it triggered a few red flags on our plagiarism checker - mostly small sentences - but a few that would benefit from changing. 🚀  
See attached
[ESLint.pdf](https://github.com/section-io/engineering-education/files/5066449/ESLint.pdf)

**Comment by cyberShaw** [comment]
> Good afternoon @cyberShaw - just starting a preliminary review on the ESLint article - it's looking great so far - but it triggered a few red flags on our plagiarism checker - mostly small sentences - but a few that would benefit from changing. 🚀
> See attached
> [ESLint.pdf](https://github.com/section-io/engineering-education/files/5066449/ESLint.pdf)

Oh, apologies. I'll make some revisions and commit them! 😄

**Comment by cyberShaw** [comment]
> Good afternoon @cyberShaw - just starting a preliminary review on the ESLint article - it's looking great so far - but it triggered a few red flags on our plagiarism checker - mostly small sentences - but a few that would benefit from changing. 🚀
> See attached
> [ESLint.pdf](https://github.com/section-io/engineering-education/files/5066449/ESLint.pdf)

@hectorkambow I just checked the plagiarism report, and match #2 to match #8 in the document were instructions on using the eslint-cli. When I initialize eslint, it asks a standard set of questions. That's why these matches were triggered. 

Screenshot attached for your reference below 😄 

image

**Comment by hectorkambow** [comment]
👋  Hi @npwaters Nathan - wanted to ask if you could give this Node using ESLint article a tech review whenever you have a moment.  🚀 
Thank you

File: articles/node-eslint/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+- [ESLint and Node](#eslint-and-node)
+- [Airbnb Style Guide](#airbnb-style-guide)
+- [Further Reading](#further-reading)
+
+### Introduction to Linting
+
+Linters are tools that perform **static code analysis** in software testing.
+
+Static Code Analysis is the analysis of computer software that is performed on static code. It is performed by looking at source code without running it. This process is usually done using a software or an automated tool. Static Code Analysis is done as a part of **Continuous Integration and Delivery**.
+
```
Comment (npwaters) [inline articles/node-eslint/index.md:34]: Static analysis is performed during the coding phase - this is before CI/CD is performed.

**Pull request merged**